product_docs/docs/migration_toolkit/55/08_mtk_command_options.mdx

```diff
@@ -570,7 +596,7 @@ The uppercase naming convention is preserved for tables, views, sequences, proce
 
 The default behavior of the Migration Toolkit without the `-useOraCase` option is that database object names are extracted from Oracle without enclosing quotation marks unless the database object was explicitly created in Oracle with enclosing quotation marks. The following is a portion of a table command generated by the Migration Toolkit with the `-offlineMigration` option:
```

Comment:
@josh-heyer , do you agree we should use sql for the code blocks in this section? Or would text be better?